shared/common-adapters/icon.shared.js

```diff
@@ -75,6 +75,8 @@ export function fontSize(type: IconType): ?Object {
 
 export function typeToFontSize(sizeType: SizeType) {
   switch (sizeType) {
+    case 'Bigger':
+      return Styles.isMobile ? 48 : 36
```

Comment:
If we can have `Huge` too that'd be great!

Reply:
Let's add that when we need it (in a relevant PR).

Reply:
(Resolved on chat that we should add it in PRs using it.)